modules/sdk-core/src/bitgo/inscription/inscription.ts

```diff
@@ -0,0 +1,84 @@
+import assert from 'assert';
+import { IInscription } from './iInscription';
+import { BitGoBase } from '../bitgoBase';
+import { IWallet } from '../wallet';
+import { inscriptions, PreparedInscriptionRevealData, TapLeafScript } from '@bitgo/utxo-ord';
+import { KeyIndices } from '../keychain';
+import { xprvToRawPrv } from '../../account-lib';
+import { HalfSignedUtxoTransaction } from '../baseCoin';
+
+export class Inscription implements IInscription {
+  private readonly bitgo: BitGoBase;
+  private readonly wallet: IWallet;
+
+  constructor(bitgo: BitGoBase, wallet: IWallet) {
+    this.bitgo = bitgo;
+    this.wallet = wallet;
+  }
+
+  async prepareReveal(
+    inscriptionData: Buffer,
+    contentType: string,
+    feeRate: number
+  ): Promise<PreparedInscriptionRevealData> {
+    const user = await this.wallet.baseCoin.keychains().get({ id: this.wallet.keyIds()[KeyIndices.USER] });
+    assert(user.pub);
+    const pubkey = Buffer.from(user.pub);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore it exists
+    const network = this.bitgo.coin(this.wallet.coin()).network;
+
+    return inscriptions.createInscriptionRevealData(pubkey, contentType, inscriptionData, network);
+  }
+
+  /**
+   *
+   * @param walletPassphrase
+   * @param tapLeafScript
+   * @param unsignedCommitTx
+   * @param commitTransactionUnspents
+   * @param commitAddress
+   * @param recipientAddress
+   */
+  async signAndSend(
+    walletPassphrase: string,
+    tapLeafScript: TapLeafScript,
+    unsignedCommitTx: string,
+    commitTransactionUnspents: any[],
+    commitAddress: string,
+    recipientAddress: string
```

Comment:
Nit: Although I'd have preferred to condense the arguments but if we're not doing that lets at least reorder the arguments to keep properties of same object next to each other.
Eg: `tapLeafScript` & `commitAddress` are part of same revealData object, similarly `unsignedCommitTx` & `commitTransactionUnspents`.